src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,119 @@
+package wolox.training.models;
+
+import javax.persistence.Entity;
+import javax.persistence.Column;
+
+@Entity
+public class Book {
```

Comment:
Add some java docs, is not part of the card but they help a lot

Reply:
I'm not sure how to do it, is this ok? I've used a plugin to generate it.

Reply:
yes, is not a bad idea use a plugin for java docs. Maybe when you have some method with a lot of business logic you should write it yourself